scripts/BilbyCustomFunctions_Reduction.py

```diff
@@ -0,0 +1,343 @@
+# Mantid Repository : https://github.com/mantidproject/mantid
+#
+# Copyright &copy; 2018 ISIS Rutherford Appleton Laboratory UKRI,
+#     NScD Oak Ridge National Laboratory, European Spallation Source
+#     & Institut Laue - Langevin
+# SPDX - License - Identifier: GPL - 3.0 +from mantid.simpleapi import *
+
+import csv
+import math
+from itertools import product
+import sys
+from mantid.simpleapi import MoveInstrumentComponent, CropWorkspace
+
+
+#######################################################################################
+# REDUCTION ###########################################################################
+#######################################################################################
+def string_boolean(line):
+    """ Convert string to boolean; needed to read "true" and "false" from the csv Data reduction settings table """
+
+    if line == 'false':
+        bool_string = False
+    elif line == 'true':
+        bool_string = True
+    else:
+        print("Check value of {}".format(line))
+        print("It must be either True or False")
+        sys.exit()
+    return bool_string
+
+
+#######################################################################################
+
+def read_convert_to_float(array_strings):
+    """ Needed to convert binning parameters from the csv file into the float numbers """
+
+    array = [x.strip() for x in array_strings.split(',')]
+    array = [float(x) for x in array]
+    if (len(array) != 3):
+        print("Check input parameters; binning parameters shall be given in a format left_value, step, right_value.")
+        sys.exit()
+    return array
+
+
+###########################################################################################
+
+def FilesListReduce(filename):
```

Comment:
Shouldn't functions in python be in snake_case unless they need to be that due to inheriting from a C++ class or the like?